### DISCUSSION TITLE
CIFAR10-DVS ISSUES

### DISCUSSION THREAD

**Discussion by rkdgmlqja** [neuromorphs/tonic#225] [Q&A]
Recently I found out that tonic's CIFAR10-DVS dataset's download link was blocked so I tried to use original CIFAR10-DVS datseth from https://figshare.com/articles/dataset/CIFAR10-DVS_New/4724671/1 but the data format is aedat which is really hard to find it's decoding process I tried https://github.com/qiaokaki/AedatTools this tool to use aedat file but it is not working. 
So  is there any way to get aedat4 formatted CIFAR10-DVS online??

**Comment by tobidelbruck** [comment]
did you try Cedric's https://github.com/SensorsINI/AedatTools that we 
forked?

**Comment by fabrizio-ottati** [comment]
Also, `loris` should work fine (I have used CIFAR10-DVS with no issues in the past but you need to install it separately from `pip` or `conda` since we do not want to depend on it 'cause it is not maintained anymore, I think (@biphasic)). Maybe I will provide a datapipe version of CIFAR10-DVS that takes care of this dependency, but it will take some time :)
Thanks for the hint, @tobidelbruck.

**Comment by biphasic** [comment]
Hello @rkdgmlqja ,
the original dataset is in aedat v2 format, which is really hard to decode. I asked the person who converted it previously to reinstate the upload https://github.com/neuromorphs/tonic/pull/172#issuecomment-1294814423

**Comment by biphasic** [comment]
@rkdgmlqja I uploaded the dataset files to a new repository and updated the urls in bc5dca2cb60954b1e4482961ab584236427e4eb5. It should work now again, please try it
